enterprise/templatetags/enterprise.py

```diff
@@ -0,0 +1,30 @@
+"""
+Template tags for enterprise apps.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django import template
+from django.utils.safestring import mark_safe
+
+
+register = template.Library()  # pylint: disable=invalid-name
+
+
+@register.simple_tag(name="fa_icon")
+def fa_icon(tags):
+    """
+    Django template tag that returns font awesome icon depending upon message tags.
+
+    Usage:
+        {% fa_icon "success" %}
+    """
+    if 'success' in tags:
+        return mark_safe('<i class="fa fa-check-circle" aria-hidden="true"></i>')
+    if 'info' in tags:
+        return mark_safe('<i class="fa fa-info-circle" aria-hidden="true"></i>')
+    if 'warning' in tags:
+        return mark_safe('<i class="fa fa-exclamation-circle" aria-hidden="true"></i>')
+    if 'error' in tags:
+        return mark_safe('<i class="fa fa-times-circle" aria-hidden="true"></i>')
```

Comment:
Instead of all these `if` statements, create a module-scoped dictionary which maps the message type name to the icon class.
```
MESSAGE_ICONS = {
    'success': 'fa-check-circle',
    'info': 'fa-info-circle',
    'warning': 'fa-exclamation-circle',
    'error': 'fa-times-circle'
}
```
And then you can do:
```
icon_class = MESSAGE_ICONS.get(message_type)
if icon_class:
    return mark_safe('<i class="fa {icon_class}" aria-hidden="true"></i>'.format(icon_class=icon_class))
else:
    return ''
```